ecosystem/typescript/sdk_v2/src/crypto/asymmetric_crypto.ts

```diff
@@ -63,23 +50,14 @@ export abstract class PrivateKey implements Serializable, Deserializable<Private
   abstract publicKey(): PublicKey;
 }
 
-/**
- * An abstract representation of a signature.  This is the product of signing a
- * message and can be used with the PublicKey to verify the signature.
- */
-export abstract class Signature implements Serializable, Deserializable<Signature> {
-  /**
-   * Get the raw signature bytes
-   */
+export abstract class Signature extends Serializable {
+  // Convert the signature to bytes or Uint8Array.
```

Comment:
Let's get this back in here, as I added doc comments

Reply:
oops that's my bad, thought I incorporated the new ones when merging